e2e_tests/test_shared_services.py

```diff
@@ -2,7 +2,7 @@
 import logging
 
 from resources.resource import disable_and_delete_resource, post_resource
-from helpers import get_shared_service_id_by_name
+from helpers import get_shared_service_by_name
```

Comment:
make sure all calls to this renamed function was changed,
line number 100 still uses `get_shared_service_id_by_name`

Reply:
Good catch, thank you!